pulp_python/tests/functional/api/test_sync.py

```diff
@@ -87,9 +87,7 @@ def test_sync(self):
         sync(self.cfg, remote, repo)
         repo = self.client.get(repo['pulp_href'])
 
-        self.assertNotEqual(latest_version_href, repo['latest_version_href'])
-        self.assertDictEqual(get_content_summary(repo), PYTHON_XS_FIXTURE_SUMMARY)
-        self.assertDictEqual(get_added_content_summary(repo), {})
```

Comment:
Why remove this?

Reply:
Because this is no longer true. Since a new 'no-op' version is not getting created, the 'added_content_summary' is no longer empty. The assertion about the 'content_summary' is already made on line 82.

Reply:
Makes sense.